.github/CONTRIBUTING.md

```diff
@@ -168,6 +168,7 @@ Notes:
   - on Ubuntu (Linux) run `apt-get install texlive` or otherwise follow the instructions on the TeXLive website
   - use the [RTD docker image](https://hub.docker.com/r/readthedocs/build)
 - with PL used class meta you need to use python 3.7 or higher
+- When you submit a PR for fixing documentation, make sure to include `[ci skip]` or `[skip ci]` in the PR name to skip unrelated unittesting CIs.
```

Comment:
I think this will go after 0.10, but rather in changelog write notes to the contribution guide :]

Reply:
Isn't it already in contribution guide?